Assets/Scripts/Menu/MenuBird.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MenuBird : MonoBehaviour
+{
+    public float speed = 2.0f;
+    public int direction = 1;
+
+    private float screenEnds = 20;
+
+    public void Start()
+    {
+        
+    }
+
+    public void Update()
+    {
+        if(direction >= 1)
```

Comment:
Could you change the `direction` to a type `Vector2` and then have the bird always translating using `direction * Time.deltaTime * speed`, then when the bird reaches the edges of the screen flip the direction between `Vector2.right` and `Vector2.left`.

Reply:
They don't turn around, they pop onto the opposite side of the screen, resetting their path. But I can improve the code slightly by using a velocity vector instead and always translating on that.